Codeinwp/neve
# Apply font-family for inputs

### Summary
We are not adding any font-family property for that text area. It's the user agent that makes it monospace. While in Chrome, all the text areas that don't have a font-family defined are monospace, in safari, the font-family is "system-ui."

What I did here is to apply font family on these selectors:
`form input:read-write, form textarea, form select, form select option, form.wp-block-search input.wp-block-search__input`
They didn't have the font-family property, and if chrome wouldn't have that user agent style, they should inherit the font-family from the body tag. That's why I think adding the property won't break anything else.

The same issue applies to search inputs. By default, they took the user's agent font family, which in chrome is "Arial." Here, I've applied the general font settings for `form.search-form input:read-write` selector.


### Will affect visual aspect of the product
YES


### Test instructions
- Check if the text area has the font family defined in the customizer.
- Check if the search input, the one in the sidebar has the font family from the customizer.
- Check If the selectors I've mentioned previously had a font-family from somewhere and now are overwritten.


Closes #2579.
